src/views/highchart.js

```diff
@@ -0,0 +1,169 @@
+        }, options);
+    }
+
+    _applyHighchartOptionsDefaults(highchartOptions) {
+        highchartOptions = highchartOptions || {};
+
+        return _.extend({
+            xAxis: _.extend({
+                type: 'datetime'
+            }, highchartOptions.xAxis),
+            chart: _.extend({}, highchartOptions.chart)
+        }, highchartOptions);
+    }
+
+    _addSeries(series) {
+        let seriesLabel = this._seriesDetector.getSeriesLabel(series.id);
+        let config = this._findMatchingSeriesConfig(series.keys);
+        seriesLabel = seriesLabel !== '' ? seriesLabel : this._attributes.valueField;
+
+        this.chart.addSeries(_.extend({
+            id: 'series-' + series.id,
+            name: seriesLabel
+        }, config));
+
+        this._series[series.id] = series;
+    }
+
+    _convertUserOptionsToAttributes(options) {
+        return _.pick(options, 'valueField', 'keyField', 'xField', 'series');
+    }
+
+    _consume(points) {
+        if (points.length === 0) {
+            return;
+        }
+
+        if (!this._attributes.valueField) {
+            this._attributes.valueField = this._findValueField(points[0]);
+        }
+
+        points.forEach((point) => {
+            let series = this._seriesDetector.getSeriesForPoint(point);
+
+            if (!this._series[series.id]) {
+                this._addSeries(series);
+            }
+
+            let xFieldValue = point[this._attributes.xField];
+
+            this.chart.get('series-' + series.id).addPoint([
+                _.isDate(xFieldValue) ? xFieldValue.getTime() : xFieldValue,
+                point[this._attributes.valueField] ]);
+        });
+    }
+
+    _findMatchingSeriesConfig(keys) {
+        let keyValues = _.values(keys);
+        let seriesConfig = {};
+        if (this._attributes.series) {
+            let seriesConfigKeys = Object.keys(this._attributes.series);
+            for(let i = 0; i < seriesConfigKeys.length; i++) {
+                let configKey = seriesConfigKeys[i];
+                let config = this._attributes.series[configKey];
+
+                if (_.contains(keyValues, configKey)) {
+                    seriesConfig = config;
+                    break;
+                }
+            }
+        }
+
+        return seriesConfig;
+    }
+
+    setDimensions(id, width, height) {
+        this.chart.setSize(width, height, false);
+    }
+
+    destroy() {
+        this.chart.destroy();
+        super.destroy();
+    }
+
+    _findValueField(point) {
+        if (point.value !== null && _.isNumber(point.value)) {
+            return 'value';
+        }
+        else {
+            let valueField;
+
+            let fields = Object.keys(point);
+
+            for(let i = 0; i < fields.length; i++) {
+                let field = fields[i];
+
+                if (field !== this._attributes.xField && _.isNumber(point[field])) {
+                    valueField = field;
```

Comment:
I changed this to use the superclass's `_determineValueField` which shows a runtime warning message if it wasn't able to find a usable field.

Using the first one that is found matches the behavior of the rest of the views. I can open a separate issue to fail if there is more than one candidate and require the user to specify a `-valueField` if thats the behavior we want.